clustered/integration-test/src/test/resources/clusteredConfiguration.txt

```diff
@@ -14,7 +28,7 @@ caches:
                     size: 1 MB
                     tierHeight: 1000
                 clustered-dedicated:
-                    size: 2 MB (persistent)
+                    size: Pool{ 2MB clustered-dedicated(persistent) from='primary-server-resource'}
```

Comment:
A clustered resource will always be marked as `persistent`, should this be suppressed from the `toString()` and `readableString()`?

Reply:
are you talking about removing 3 lines from : https://github.com/ehcache/ehcache3/blob/a25d02c4dcb5744c6403447237a23e7ac93c7e01/clustered/client/src/main/java/org/ehcache/clustered/client/internal/config/DedicatedClusteredResourcePoolImpl.java#L83 ?

Reply:
yes

Reply:
ok